app/controllers/line_items_controller.rb

```diff
@@ -66,13 +67,14 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_line_item
-      @line_item = LineItem.find(params[:id])
-    end
+  # Use callbacks to share common setup or constraints between actions.
+  def set_line_item
+    @line_item = LineItem.find(params[:id])
+  end
 
-    # Never trust parameters from the scary internet, only allow the white list through.
-    def line_item_params
-      params.require(:line_item).permit(:food_id)
-    end
+  # Never trust parameters from the scary internet, only allow the white list through.
+  def line_item_params
```

Comment:
redirect or render difference, and what happens in RecordNotFound Exception--- redirect or render